viadot/sources/business_core.py

```diff
@@ -141,18 +141,16 @@ def to_df(self, if_empty: str = "skip") -> pd.DataFrame:
             APIError: When selected API view is not available.
         """
         view = self.url.split("/")[-1]
+
         if view not in ["GetCustomerData", "GetItemMaster", "GetPendingSalesOrderData"]:
             raise APIError(f"View {view} currently not available.")
-        if view in ("GetCustomerData", "GetItemMaster"):
-            data = self.get_data().get("MasterDataList")
-            df = pd.DataFrame.from_dict(data)
-            logger.info(
-                f"Data was successfully transformed into DataFrame: {len(df.columns)} columns and {len(df)} rows."
-            )
-            if df.empty:
-                self._handle_if_empty(if_empty)
-            return df
-
-        if view == "GetPendingSalesOrderData":
-            # todo waiting for schema
-            raise APIError(f"View {view} currently not available.")
+
+        data = self.get_data().get("MasterDataList")
+        df = pd.DataFrame.from_dict(data)
+        logger.info(
+            f"Data was successfully transformed into DataFrame: {len(df.columns)} columns and {len(df)} rows."
```

Comment:
Do we need information about columns and rows?

Reply:
I think it might be useful when ie API structure changes and its clear for the user that sometting is wrong